Tensile/KernelWriterAssembly.py

```diff
@@ -7409,6 +7473,140 @@ def shiftVectorComponents(self, kernel, tP):
     self.vgprPool.checkIn(vReg)
     return kStr
 
+  def shiftVectorComponentsForMatrixInst(self, kernel, tP):
+    kStr = ""
+
+    # glvw
+    glvw = tP["glvw"]
+
+    numThreadInWave            = 64
+    numContinuousOutput        = 4
+    numOutputThreads1          = kernel["MatrixInstN"]
+    numOutputThreads0          = numThreadInWave // numOutputThreads1
+    numSubOutputPerWave0       = numOutputThreads0 * numContinuousOutput
+    numSubOutputGroupsPerWave0 = kernel[tP["mt"]] // numSubOutputPerWave0
+    numShiftBlock              = numContinuousOutput // glvw
+    numOutputElements          = numSubOutputGroupsPerWave0 * numContinuousOutput
+
+    subTile1 = (kernel["MacroTile1"] // ((kernel["SubGroup0"] * kernel["SubGroup1"]) // numThreadInWave)) // kernel["MatrixInstN"]
+
+    # labels
+    svrLabels = []
+    sviLabels = []
+    svoLabels = []
```

Comment:
Comment defining svr, svi, svo could help

Reply:
update comment and make a slide to explain in e-mail